docs/packages.md

```diff
@@ -172,6 +172,29 @@ pkg:
   content-trust-passphrase-command: "lpass show <key> --password"
 ```
 
+#### Signing Manually
+
+If, for whatever reason, you want to sign an individual tag manually, whether the index (a.k.a. "multi-arch manifest") or the architecture-specific manifest, do the following:
+
+1. Make sure you have ready your credentials:
+   * docker hub login and passphrase
+   * docker notary signing key passphrase
+1. Set an env var with the name of the image **not** including tag. Make sure to include the registry host. For example, `IMAGE=docker.io/linuxkit/containerd`
+1. Set an env var with the tag. For example, `TAG=a4aa19c608556f7d786852557c36136255220c1f`. This example is a multi-arch manifest tag, but you could just as easily sign any other tag, such as the tag of an individual arch-specific image, `TAG=a4aa19c608556f7d786852557c36136255220c1f-s390x`.
+1. Set an env var with the size of the artifact that is pointed to by the tag, in bytes. If the tag points to an index, then it is the size of the index; if it is a specific manifest, then it is the size of the manifest. For example, `SIZE=1052`.
```

Comment:
it would be good to expand on how to get this size. I used the manifest-tool for this, but maybe some `docker image inspect | jq` magic does this too. I remember a few years back I struggled a lot to get this size value

Reply:
I actually cheat. I have my own utility that retrieves this. I will add general documentation and reference to my own utility. `docker image inspect` does not provide this IIRC.